src/zzz_od/gui/app_setting/life_on_line_setting_flyout.py

```
@@ -51,7 +51,9 @@ def init_config(self) -> None:
         )
 
         self.times_opt.init_with_adapter(get_prop_adapter(config, 'daily_plan_times'))
-        self.done_value.titleLabel.setText(f'当日: {run_record.daily_run_times}')
+        self.done_value.titleLabel.setText(
+            gt('当日: {count}', count=run_record.daily_run_times)
```

Ruff (0.16.1)

[error] Undefined name `gt`

(F821)